libs/judicial-system/types/src/lib/tables/caseTableGroup.ts

```diff
@@ -226,5 +216,9 @@ export const getCaseTableGroups = (
     return publicProsecutorsOfficeTableGroups
   }
 
+  if (isPublicProsecutionUser(user)) {
```

Comment:
Differenct prosecution roles should see different table groups. Prosecutor representatives onsly see indictment cases.

Reply:
Yes, lets merge the prosecutor police lists into this one and then merge to main